### PR TITLE
Update pull_request_template

### PR DESCRIPTION
**Pull request by zhouqiang-cl**
Signed-off-by: zhouqiang <zhouqiang.cl@gmail.com>

###  What have you changed?

Please explain in detail what the changes are in this PR and why they are needed:
It changes the pull request template. No code

###  What is the type of the changes?

Pick one of the following and delete the others:
- Misc (other changes)

###  How is the PR tested?
- unit test
###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?
No

###  Does this PR affect `tidb-ansible`?
No

###  Refer to a related PR or issue link (optional)
No
###  Benchmark result if necessary (optional)

No
###  Any examples? (optional)
No

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
@breeswish PTAL

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -28,8 +28,11 @@ Pick one of the following and delete the others:
 - Misc (other changes)
 
 ###  How is the PR tested?
-
-Please describe the tests that you ran to verify your changes. Have you added unit or integration tests, or run manual tests? What additional tests would give you greater confidence in this change?
+Pick some of the following tests which be tested
```
Comment (dcalvin) [inline .github/pull_request_template.md]:
```suggestion
Please select the tests that you ran to verify your changes:
```

Comment (Connor1996) [inline .github/pull_request_template.md]: The description is a little weird, but I have no idea with it. What's your opinion? @dcalvin

**Comment by dcalvin** [comment]
Also, please change the line: 
> If there is a document change, please file a PR in ([docs](https://github.com/tikv/tikv/tree/master/docs)) and add the PR number here.

with the current doc repo link: <https://github.com/tikv/website/tree/master/content>

File: .github/pull_request_template.md (hunk 1 above)
Comment (dcalvin) [reply .github/pull_request_template.md]: Already suggested a change above.

**Comment by zhouqiang-cl** [comment]
@Connor1996 @dcalvin  PTAL again

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -28,12 +28,15 @@ Pick one of the following and delete the others:
 - Misc (other changes)
 
 ###  How is the PR tested?
-
-Please describe the tests that you ran to verify your changes. Have you added unit or integration tests, or run manual tests? What additional tests would give you greater confidence in this change?
+Please select the tests that you ran to verify your changes:
+- Unit test
+- Integration test
+- Manual test (add detailed scripts or steps below)
+- No code
```
Comment (dcalvin) [inline .github/pull_request_template.md]: What does "no code" mean?

Comment (zhouqiang-cl) [reply .github/pull_request_template.md]: No code means did not have any code. then did not need to test

Comment (zhouqiang-cl) [reply .github/pull_request_template.md]: the same with tidb

**Pull request merged**